sites/skeleton.dev/src/routes/(inner)/components/listboxes/+page.svelte

```diff
@@ -113,6 +114,46 @@
 	<ListBoxItem bind:group={valueMultiple} name="medium" value="books">Books</ListBoxItem>
 	<ListBoxItem bind:group={valueMultiple} name="medium" value="movies">Movies</ListBoxItem>
 	<ListBoxItem bind:group={valueMultiple} name="medium" value="tv">TV</ListBoxItem>
+</ListBox>
+			`}
+					/>
+				</svelte:fragment>
+			</DocsPreview>
+		</section>
+		<section class="space-y-4">
+			<h3 class="h3">Disabled</h3>
+			<p>
+				By adding the <code class="code">disabled</code> property, 
```

Comment:
using the one prop approach, this should be delete because it is easy to understand and doesn't require additional description.
look at https://www.skeleton.dev/components/tree-views#disabled for example

Reply:
I've narrowed down the description according to the example given